algorithms/JavaScript/README.md

```diff
@@ -22,3 +22,10 @@
 ## Strings
 - [Palindrome](src/strings/palindrome.js)
 - [Sequence](src/strings/sequence.js)
+
+## Stack
+- [Stack Implementation](src/stack/stackImpl.js)
+- [Stack Using Two Queue](src/stack/usingTwoQueue.js)
+
+## Queue
+- [Queue Implementation](src/Queue/queueImpl.js)
```

Comment:
Hi @underager,
Please rename the folder name to plural and lowercase and the file name as a suggestion.
```suggestion
## Stack
- [Stack Implementation](src/stacks/stack.js)
- [Stack Using Two Queue](src/stacks/two-stack.js)

## Queue
- [Queue Implementation](src/queues/queue.js)
```

Reply:
Update the reference @underager